docs/faq.md

```diff
@@ -110,7 +110,7 @@ if it even tries to pull an image from a remote server.
 Docker for building container images.
 
 Tilt will automatically enable Buildkit if your local Docker installation
-supports it.
+supports it and if the `DOCKER_BUILDKIT` environment variable is **not** set to `0`.
```

Comment:
I might phrase this as "if you want to disable buildkit manually, set DOCKER_BUILDKIT=0"

to avoid the negative